web/elm/src/Views/FavoritedIcon.elm

```diff
@@ -0,0 +1,35 @@
+module Views.FavoritedIcon exposing (view)
```

Comment:
It would be nice if we reused this module in each place we display the favourite icon (i.e. also the sidebar and the pipeline cards).

Maybe we could extend `params` to also take in:

* `sizePx` (20 for topbar and pipeline card, and 18 for sidebar)
* `domID` (in the same way we did for the `PauseToggle` in https://github.com/concourse/concourse/pull/5922/commits/33d94d354f672c1b0d0048190ada04fc64eee794)
* `margin` (`17px` for topbar, 0 for sidebar and pipeline card) - or alternatively, we could pass in `styles : List Html.Attribute msg` that we extend the default list of styles with, which is a lot more flexible